CAPE/CAPE/run_cape_offline.cpp

```diff
@@ -170,8 +171,6 @@ int main(int argc, char ** argv){
 
     cv::Mat_<float> X(height,width);
     cv::Mat_<float> Y(height,width);
-    cv::Mat_<float> X_t(height,width);
```

Comment:
`X_t` and `Y_t` were needed in order to combine RGB image and Segmented image together and then visualize them in one frame. We don't need visualization and that's why we don't need these variables.